client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -180,7 +180,7 @@ class CancelPurchase extends React.Component {
 		}
 
 		if ( isSubscription( purchase ) ) {
-			heading = this.props.translate( 'Cancel Your %(purchaseName)s Subscription', {
+			heading = this.props.translate( 'Cancel your %(purchaseName)s Subscription', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.